doc/getting_started/getting_started.rst

```diff
@@ -38,19 +40,19 @@ Download MoveIt 2 Tutorials Source
 Move into your colcon workspace and pull the MoveIt 2 tutorials source: ::
 
   cd ~/ws_moveit2/src
-  git clone https://github.com/ros-planning/moveit2_tutorials.git
+  git clone https://github.com/ros-planning/moveit2_tutorials.git -b foxy
```

Comment:
Maybe not a problem of this PR but we should remind people to source their ROS2 installation (which seems to be a common problem among newcomers) `source /opt/ros/foxy/setup.bash`